internal/backend/oauth/webhook.go

```diff
@@ -103,6 +103,25 @@ func (s *svc) exchange(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Special case: GitHub gives us what we need for generating JWTs in the future
+	// (i.e. the GitHub app's installation ID) without exchanging the code below.
+	if intg == "github" {
```

Comment:
ugh, i wonder if there is a way to put this code into the github module or generalize this somehow.

Reply:
There are 3 options, none of them makes sense, at least now:

1. Move all the token exchange logic to each integration's handler - violates the DRY principle in a big way
2. Keep this function as a shared function, but not a webhook, and call it in all integrations except GitHub - still feels like a flavor of (1), and will require reconfiguration of all existing OAuth apps which is very disruptive
3. Expose a token-exchange-customization function in each integration, but implement only in GitHub - feels like overkill if only GitHub needs it